recipes/gnali/meta.yaml

```diff
@@ -33,13 +33,19 @@ requirements:
     - git >=2
   run:
     - python >=3.6
+    - ensembl-vep ==101.*
```

Comment:
why do you need to specify `101.*`

Reply:
Updated as per our discussion in 53449d9